index.html

```diff
@@ -0,0 +1,453 @@
+            class="card-img desk"
+            id="img"
+            src="images/section-work/snapshoot4.png"
+            alt="Tonic"
+          />
+          <div class="desk-pop">
+            <div class="first">
+              <p class="text-secondary mob desk" id="description"></p>
+            </div>
+            <div class="second">
+              <ul class="card-tags desk" id="tags"></ul>
+              <div class="line"></div>
+              <div class="popup-button desk">
+                <a href="#" class="button btn-element desk"
+                  >See Live
+                  <img src="images/section-work/Live-icon.svg" alt="live"
+                /></a>
+                <a href="#" class="button btn-element desk"
+                  >See Source
+                  <img src="images/section-work/Github-icon.svg" alt="git"
+                /></a>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+     POPUP END -->
+
+
+    <nav class="navigation">
+      <p class="logo">My Logo</p>
+      <img src="images/Union.svg" alt="logo" class="hamburger open-menu" />
+      <ul class="item-navigation">
+        <li><a href="#work">Portfolio</a></li>
+        <li><a href="#about">About</a></li>
+        <li><a href="#contact">Contact</a></li>
+      </ul>
+    </nav>
+
+<!--
+    <div class="overlay hidden">
+      <button type="button" class="close-overlay close-menu">
+        <img src="images/close-overlay.svg" alt="close-overlay" />
+      </button>
+
+      <ul class="sections-list">
+        <li><a href="#work" class="option">Portfolio</a></li>
+        <li><a href="#about" class="option">About</a></li>
+        <li><a href="#contact" class="option">Contact</a></li>
+      </ul>
+    </div>
+-->
+
+    <header class="header">
+      <div class="header-text">
+        <h1 class="heading-primary">I'm Ellis<br> Glad to see you!</h1>
+        <p class="text-primary">
+          I’m a software developer! I can help you build a product , feature or
+          website Look through some of my work and experience! If you like what
+          you see and have a project you need coded, don’t hestiate to contact
+          me.
+        </p>
+        <a href="#" class="connect-btn">LET'S CONNECT</a>
+        <ul class="social-icons">
+          <li>
+            <a href="#"><img src="./images/Twitter.svg" alt="Twitter" /></a>
+          </li>
+          <li>
+            <a href="#"
+              ><img src="./images/Linkedin-icon.svg" alt="Linkedin-icon"
+            /></a>
+          </li>
+          <li>
+            <a href="#"><img src="./images/Medium.svg" alt="Medium" /></a>
+          </li>
+          <li>
+            <a href="#"><img src="./images/GitHub.svg" alt="GitHub" /></a>
+          </li>
+          <li>
+            <a href="#"><img src="./images/Vector.svg" alt="Vector" /></a>
+          </li>
+        </ul>
+      </div>
+    </header>
+
+    <section class="work" id="work">
```

Comment:
It's a better idea to style this section with display of grid. Grid will enable the items to respond to any screen size hence making the page responsive.  Please make that change accordingly.

Reply:
thank you will do just that